File: liqp-core/src/main/java/liqp/tags/Continue.java
package liqp.tags;

import liqp.tag.LTag;
import liqp.node.LNode;
import liqp.context.LContext;

public class Continue extends LTag {

    @Override
    public Object render(LContext context, LNode... nodes) {
        return CONTINUE;
    }
}

File: liqp-core/src/main/kotlin/liqp/tags/Continue.kt
package liqp.tags

import liqp.ControlResult.CONTINUE
import liqp.context.LContext
import liqp.node.LNode
import liqp.tag.LTag

class Continue : LTag() {

  override fun render(context: LContext, vararg nodes: LNode): Any? {
    return CONTINUE
  }
}
